src/game_api/state_structs.hpp

```diff
@@ -737,19 +751,22 @@ struct LiquidPhysicsParams
 
 struct LiquidPhysics
 {
-    size_t unknown1;
-    union
-    {
-        std::array<LiquidPhysicsParams, 5> pools;
-        struct
-        {
-            LiquidPhysicsParams water_physics;
-            LiquidPhysicsParams coarse_water_physics;
-            LiquidPhysicsParams lava_physics;
-            LiquidPhysicsParams coarse_lava_physics;
-            LiquidPhysicsParams stagnant_lava_physics;
-        };
-    };
+    size_t unknown1; // MysteryLiquidPointer1 in plugin
```

Comment:
Those three vals could be in a struct and then we can put this stuff back into the same union, just putting that struct instead of LiquidPhysicsParams.

Reply:
yeah, but why? is there a need to do it that way?